lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -19,51 +23,55 @@ import '../semantics.dart' show EngineSemanticsOwner;
 /// It also takes into account semantics being enabled to fix the case where
 /// offsetX, offsetY == 0 (TalkBack events).
 ui.Offset computeEventOffsetToTarget(DomMouseEvent event, DomElement actualTarget) {
-  // On top of a platform view
-  if (event.target != actualTarget) {
-    return _computeOffsetOnPlatformView(event, actualTarget);
-  }
   // On a TalkBack event
   if (EngineSemanticsOwner.instance.semanticsEnabled && event.offsetX == 0 && event.offsetY == 0) {
     return _computeOffsetForTalkbackEvent(event, actualTarget);
   }
+
+  // On one of our text-editing nodes
+  final bool isInput = textEditing.strategy.domElement?.contains(event.target! as DomNode) ?? false;
```

Comment:
Just a sanity check - this will always be false because `event.target` is either the glass pane or a platform view pre-restructure of the DOM?

Reply:
event.target is always the glass-pane before the restructure of the DOM, so it'll never be a child of the `domElement` on the text strategy.

(Note that in a post-restructure world, we could use this same check I think!)